rules/xp.json

```diff
@@ -186,7 +186,22 @@
       "28": 25,
       "29": 25,
       "30": 25,
-      "31": 27
+      "31": 27,
+      "32": 27,
+      "33": 27,
+      "34": 27,
+      "35": 30,
+      "36": 30,
+      "37": 30,
+      "38": 30,
+      "39": 32,
+      "40": 32,
+      "41": 32,
+      "42": 32,
+      "43": 35,
+      "44": 35,
+      "45": 35,
+      "46": 35
```

Comment:
Looks like you've got access to the Alej Number Generator :p

Reply:
I think the generator had a primary mechanism implemented with _Eyeballing_ 😉